README.md

```diff
@@ -1,14 +1,19 @@
-# Default starter for Cherry On Tech's Gridsome website
+# Welcome to Cherry On Tech's website
 
-This is the project you get when you run `gridsome create new-project`.
+We are a tech squad! We support each other as we learn and grow in our tech careers. We got the idea from Mo Hampton's JuneteenthConf talk, "Breaking into Tech." Learn more about us in an upcoming blog post!
```

Comment:
want to add the link to the talk? https://www.youtube.com/watch?v=2N5fI5j-IIs

Reply:
Good call, will do.